config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get "books/index"
+  get "books/show"
```

Comment:
Aqui seria bom a gente seguir com o padrão gerado pelo uso de `resources`, que automaticamente vai gerar as 7 actions no controller (RESTful).

```suggestion
resources: :books
```

Esse tópico aqui, a gente vai querer se aprofundar no curso, até o final dessa semana quero destrinchar essa parte no README do tópico 4 de rotas e controllers

Reply:
Beleza, vou deixar em aberto por enquanto, e fica a dúvida, acho melhor deixar a branch fora da main né? Não sei se chegamos até essa parte logo na primeira aula, o que acha?

Reply:
Boa, faz sentido! Nesse caso, eu abro o PR apontando pra `create-controller-view`

Reply:
Combinado!